install.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/sh
+
+function install_fail() {
+    echo "Installation failed"
+    exit 1
+}
+
+
+function Help(){
+    echo "Usage: install.sh [option]"
+    echo "Options:"
+    echo "  -h: Show this help message and exit"
+    echo "  -d: Install only dependencies"
+    echo "  -p: Install only python dependencies (including playwright)"
+}
+
+
+
+while getopts ":hydp" option; do
+    case $option in
+        h)
+            Help
+            exit 0;;
+        y)
+            ASSUME_YES=1;;
+        d)
+            DEPS_ONLY=1;;
+        p)
+            PYTHON_ONLY=1;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            Help
+            exit 1;;
+        :)
+            echo "Option -$OPTARG requires an argument." >&2
+            Help
+            exit 1;;
+    esac
+done
+
+
+
+function install_macos(){
+    if [ -z "$(command -v brew)" ]; then
+        NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        if [[ uname -m == "x86_64" ]]; then
+            echo "export PATH=/usr/local/bin:$PATH" >> ~/.bash_profile && source ~/.bash_profile
```

Comment:
Mac ships with zsh and bash, but uses zsh by default. I think we need to check for that. I already have a code sample for that 😉
```
# Get user's shell
case "${SHELL}" in
  */bash*)
    if [ -d "$HOME/.bash_profile" ]; then
      profile_dir="$HOME/.bash_profile"
    else
      profile_dir="$HOME/.bashrc"
    fi
    ;;
  */zsh*)
    if [ -d "$HOME/.zprofile" ]; then
      profile_dir="$HOME/.zprofile"
    else
      profile_dir="$HOME/.zshrc"
    fi
    ;;
  *)
    echo "Unknown shell, script only supports zsh and bash"
    echo "Change shell and rerun script"
    exit 0
    ;;
esac
```

Reply:
I thinks it's not needed, since Brew brew will just need the prefix once. If bash is the default shell, they prob know how to change to bash. That said, i will change the ./bash-profile (the reson i'm checking for m1 and x86 is for where brew installs)